### DISCUSSION TITLE
Can Friesach be installed on GrandOrgue?

### DISCUSSION THREAD

**Discussion by larspalo** [GrandOrgue/grandorgue#588] [Help]
Originally posted 2020-02-27 23:58:25.839000 by djsorg2062 in GrandOrgue discussion forum on SourceForge<br><br>I just downloaded Friesach for GrandOrgue on a Mac. It is a .rar file not a .organ. How can I install it? When trying to Save it, it is grayed out. I was able to Save the Burea Church from the same folder without any issue.
<br><br>Attachments in original post:<br>https://sourceforge.net/p/ourorgan/discussion/962125/thread/a440bbd20b/1e57/attachment/Screen%20Shot%202020-02-27%20at%206.56.29%20PM.png<br>

**Comment by larspalo** [comment]
Originally posted 2020-02-29 06:03:27.699000 by [Oleg Samarin](https://github.com/oleg68) in GrandOrgue discussion forum on SourceForge<br><br>.rar file - is an archive file. You have to unpack it somewhere before installing in GO

**Reply by larspalo** [reply]
Originally posted 2020-02-29 21:35:18.230000 by djsorg2062 in GrandOrgue discussion forum on SourceForge<br><br>Thanks, I was able to to unzip it by double clicking on the .rar file.

&gt; On Feb 29, 2020, at 1:03 AM, Oleg Samarin &lt;oleg68@users.sourceforge.net&gt; wrote:
&gt; 
&gt; .rar file - is an archive file. You have to unpack it somewhere before installing in GO
&gt; 
&gt; Can Friesach be installed on GrandOrgue? &lt;https://sourceforge.net/p/ourorgan/discussion/962125/thread/a440bbd20b/?limit=25#0516&gt;
&gt; Sent from sourceforge.net because you indicated interest in https://sourceforge.net/p/ourorgan/discussion/962125/ &lt;https://sourceforge.net/p/ourorgan/discussion/962125/&gt;
&gt; To unsubscribe from further messages, please visit https://sourceforge.net/auth/subscriptions/ &lt;https://sourceforge.net/auth/subscriptions/&gt;

<br><br>Attachments in original post:<br>https://sourceforge.net/p/ourorgan/discussion/962125/thread/a440bbd20b/0516/bbe9/attachment/alternate<br>

**Reply by 5763-6146** [reply]
I have Friesach on GO now installed on 2 of my Win 10 test machines. Here is a table of the RAM size requirements, as measured on my 32 Gig machine:

                                                

16 Bit                                 5.98 G     Mono                  11.86 G     Stereo 
24 Bit                                 8.93 G     Mono                  17.75 G    Stereo   

Based on these measurements, I set up Friesach/GO on my old/small/cheap i5 HP 570-p026 machine using the 16-bit mono setting above, and it plays well. Memory usage is high (82%) due to all the other stuff I have on that machine, but processor usage is only 2-3% when playing. For all tests, I play a Midi file of Rondeau (Masterpiece Theater Theme) with lots of stops enabled. It sounds magnificent. The reason I did this is that the small cheap fanless Win 10 machine in my actual organ (a converted Schober Recital from the 1970's) only has 8 G of RAM and can't be upgraded. I thus wanted to decide if I will need a new PC. It turns out I don't. 

I must give a Huge Thank You! to Lars and all the gang working on this for giving us GrandOrgue. 
I must give a similar Huge Thank You! to Piotr for giving us Friesach.

I am a long-retired EE, who cannot yet read music. My organ conversion project and the 2 pieces of software above have and continue to provide a lot of pleasure.

Cheers, Steve

**Reply by sandstone42** [reply]
I was able to use the 24 bit stereo version on my 16 Gb machine by using "Lossless Compression."

**Reply by 5763-6146** [reply]
Excellent info!  Were you using Win 10?  What was the GO/Freisach memory use?  This will be a another good data point for the table. Thanks!

**Reply by sandstone42** [reply]
Yes, I am using Windows 10. The ram usage is 12076 MB.

**Reply by 5763-6146** [reply]
Thanks! That will be very helpful for everyone, including me. I have a spare 12 G machine that is easy to upgrade to 16 G. I think that Windows 10 itself uses about 2G. I'll update the table over the weekend.

**Reply by 5763-6146** [reply]
sandstone42:  I fired up my 12G G machine and got the following additional data:
16 bit Mono Lossless 3.3 GB
24 Bit Mono Lossless 6.24 GB 
24 Bit Stereo Lossless 12.076 GB (your data)

Thanks again
